### DISCUSSION TITLE
Adding characters to Digits

### DISCUSSION THREAD

**Discussion by PlasmaHH** [Textualize/textual#5307] [Q&A]
Hi, is there any "offical" way (or planned to add it) to extend the character Digits widgets can render? I can easily see how not many other letters etc. are well enough defined to look good but it would be really useful in my case to add a few things (e.g. {} looks nice here with the appropriate utf8 chars)... I use this as a multimeter display and cobbled together something that looks nice enough to be  V ;)

I did it this way:

```python
textual.renderables.digits.DIGITS += "V"
textual.renderables.digits.DIGITS3X3 += [ .... ]
```

but that seems like its really not part of the official interface ^^
Maybe there could be some extra lookup structures where it can additionally look and use the built in as fallback?

**Comment by willmcgugan** [comment]
Nothing planned, but it isn't out of the question.

Curious how you got a V to look like a V.

**Reply by PlasmaHH** [reply]
Well its not the best but these are the utf8 chars I used:

``` python                   
textual.renderables.digits.DIGITS3X3 += [ "   ", "╲ ╱", " ᘁ " ]
```

it looks like the third doesnt render well at many places, but its kind of a small v but on the top. It is [U+1601]  CANADIAN SYLLABICS CARRIER KK and at least in my inconsolata version looks okay enough to not raise any questions fromm anyone so far ;)
